odh/base/monitoring/overrides/grafana-operator/base/grafana.yaml

```diff
@@ -15,9 +16,12 @@ spec:
         users:
             allow_sign_up: false
             auto_assign_org: true
-            auto_assign_org_role: Admin
+            auto_assign_org_role: Viewer
+            viewers_can_edit: true
         auth.anonymous:
-            enabled: false
+            enabled: true
+            org_name: Main Org.
```

Comment:
Can we name this more fitting? So it really tells this is an anonymous user? It's just a nit pick.. :slightly_smiling_face: I leave this up to you

Reply:
I think this is the default group that grafana creates, but let me test

Reply:
Yeah here we are not creating a new org, we are assigning the anon user to the existing default `Main org.`

Reply:
I just removed it :+1